script.js

```diff
@@ -4,6 +4,12 @@ const addBtn = document.getElementById('add-btn');
 const libraryCollection = document.getElementById('library');
 const bookForm = document.getElementById('addBook-form');
 const dash = document.getElementById('dash');
+const sectionBookList = document.getElementById('book-list');
+const sectionNewBook = document.getElementById('new-book');
+const contactSection = document.getElementById('Contact-section');
+const listLink = document.querySelector('.list-link');
+const newLink = document.querySelector('.new-link');
+const contactLink = document.querySelector('.contact-link');
 
 let Booklist = []; // array to store books
```

Comment:
- [ ] (optional) It would help your project portfolio if you couyld fix your app so that the correct book is being deleted from your book list.
![image](https://github.com/over-geek/Awesome-books/assets/81932478/c8bde695-645c-43dd-a861-3a69579f9e30)
As you can see from the screenshot the book I have deleted  remains in localstorage, `0: {id: 3, title: "book 4", author: "a 4"}`, and upon refresh replaces the book which was not deleted but ocuppies the index.

After Refresh
![image](https://github.com/over-geek/Awesome-books/assets/81932478/d0a75b60-9090-4acc-aae9-6e912e998108)